### PR TITLE
Bold names of players that got a point in scoreboard

### PR DESCRIPTION
**Pull request by taahamahdi**

### CODE REVIEW

**Comment by taahamahdi** [comment]
Screenshot from 2021-06-21 20-07-28

**Comment by taahamahdi** [comment]
Screenshot from 2021-06-21 20-07-56
Not as obvious with the smaller scoreboard

File: src/structures/scoreboard.ts [hunk 1]
```diff
@@ -53,12 +53,12 @@ export default class Scoreboard {
     }
 
     /** @returns An array of DiscordEmbed fields representing each participant's score */
-    getScoreboardEmbedFields(): Array<{ name: string, value: string, inline: boolean }> {
+    getScoreboardEmbedFields(roundResultIDs?: Set<string>): Array<{ name: string, value: string, inline: boolean }> {
```
[the remark this answers is not in the document]
Comment (Brainicism) [reply src/structures/scoreboard.ts:56]: Should `roundResultIDs` be `roundWinnerIDs`?

**Review by Brainicism — CHANGES_REQUESTED** [review]

File: src/structures/scoreboard.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (taahamahdi) [reply src/structures/scoreboard.ts:56]: Yeah that sounds much better, thanks

**Comment by Brainicism** [comment]
Can we get an updated picture of what the scoreboard looks like

**Comment by taahamahdi** [comment]
Screenshot from 2021-07-09 22-51-21
Screenshot from 2021-07-09 22-51-38
Screenshot from 2021-07-09 22-51-53

**Review by Brainicism — APPROVED** [review]

**Pull request merged**